reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -3124,29 +3124,37 @@ public final T blockLast(Duration timeout) {
 	}
 
 	/**
-	 * Propagate a new {@link Context} given an eventual older parent {@link Context}.
-	 * If the returned {@link Context} is empty, the propagation will be halted.
-	 * <p>
-	 *     Lifecycle for {@link Context} propagation is as such :
-	 *     <ol>
-	 *     <li> During right-to-left subscribe(Subscriber) phase, contextMap will
-	 *     read the target {@link Subscriber} context if any and cache it.</li>
-	 *     <li> Before left-to-right onSubscribe(Subscription), {@link Context}
-	 *     might be propagated from parent. If this happens, the given
-	 *     {@link Function} will be invoked with this new {@link Context}
+	 * Enrich a potentially empty upstream {@link Context}, producing a new {@link Context}
+	 * that is propagated downstream. If the returned {@link Context} is empty, the
+	 * propagation will be halted. The method is potentially invoked twice.
```

Comment:
No i should be removed, but it might be invoked more than once (not exactly twice).
I'm looking at exactly once tho, but thats a separate discussion, for now removing that line altogether is just fine.